disnake/guild.py

```diff
@@ -3350,6 +3352,57 @@ async def query_members(
             self, query=query, limit=limit, user_ids=user_ids, presences=presences, cache=cache
         )
 
+    async def search_members(
+        self,
+        query: str,
+        *,
+        limit: int = 1,
+        cache: bool = True,
+    ):
+        """|coro|
+
+        Retrieves members that belong to this guild whose username or nickname starts with
+        the query given.
+
+        Note that unlike :func:`query_members`, this is not a websocket operation.
```

Comment:
```suggestion
        Note that unlike :func:`query_members`, this is not a websocket operation, but an HTTP operation.
```
might be nice to clarify this in case anyone assumes it's a local operation